src/coreclr/.nuget/Microsoft.NET.Sdk.IL/Microsoft.NET.Sdk.IL.pkgproj

```diff
@@ -15,5 +15,24 @@
     </File>
   </ItemGroup>
 
+  <Target Name="ReplaceTemplateParametersInILTargetsTemplate"
+          Inputs="Microsoft.NET.Sdk.IL.targets.template"
```

Comment:
Same here. Put this into a property and replace the two hardcodes

Reply:
Done